timeseries/src/autogluon/timeseries/predictor.py

```diff
@@ -1048,3 +1058,12 @@ def refit_full(self, model: str = "all", set_best_to_refit_full: bool = True) ->
                     f"Training may have failed on the refit model. AutoGluon will default to using '{model_best}' for predict()."
                 )
         return refit_full_dict
+
+    def __dir__(self) -> List[str]:
+        # This hides method from IPython autocomplete, but not VSCode autocomplete
+        deprecated = ["score"]
+        return [d for d in super().__dir__() if d not in deprecated]
+
+    @Deprecated(min_version_to_warn="1.0", min_version_to_error="1.0", new="evaluate")
```

Comment:
so it both warns and errors?

Reply:
It just errors in this case. This decorator comes with functionality for both warning & raising errors. 

Actually, I think it would be cleaner if we just raise the ValueErrors ourselves, since there is no warning necessary for the breaking changes. This way we'll be certain that the exceptions are raised correctly.

Reply:
I rather meant, if we're just going to error why do we still have warn as a kwarg?

Reply:
I've updated the PR to not use this decorator, so this shouldn't be a problem anymore.

I think the motivation for the decorator was to introduce DeprecationWarnings that are later replaced by breaking changes. This isn't our use case - we just want to break everything immediately.